core.py

```
@@ -0,0 +1,431 @@
+            def connection_made(self, transport):
+                self.transport = transport
+                sock = transport.get_extra_info('socket')
+                sock.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+                sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+                
+                discovery_message = json.dumps({'type': 'discover'}).encode('utf-8')
+                
+                try:
+                    self.transport.sendto(discovery_message, ('127.0.0.1', discovery_port))
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)